### ISSUE TITLE
Error loading class 'solr.extraction.ExtractingRequestHandler'

### ISSUE THREAD

**Issue by genebobmiller** [wodby/solr#5]
When attempting to index file attachments using the Drupal module 'search_api_attachments', it was discovered that the following error is being thrown inside the solr container:

Error loading class 'solr.extraction.ExtractingRequestHandler'

I believe this is caused by the path to apache-solr-cell-\d.*\.jar or /extraction/lib being incorrect in the solrconfig.xml

**Comment by csandanov** [comment]
Does it work when you change it to `/opt/solr/contrib/extraction/lib/`?

**Comment by genebobmiller** [comment]
No that didn't work, I found that there was no apache-solr-cell.jar in the 
`/opt/solr/contrib/extraction/lib` folder.

What did work was replacing line 90 of the solrconfig.xml with the following:
```
<lib dir="/opt/solr/dist/" regex="solr-cell-7.5.0.jar" />
```

**Comment by genebobmiller** [comment]
Anything I can do to assist further in moving this forward?

**Comment by csandanov** [comment]
I see there's the line that seems to be the same thing as what you replaced with:
```
  <lib dir="${solr.install.dir:../../../..}/dist/" regex="solr-cell-\d.*\.jar" />
```

Maybe there's something wrong with `${solr.install.dir:../../../..}` but I doubt, because it used everywhere and it somehow works.

**Comment by genebobmiller** [comment]
I see this line
```
<lib dir="${solr.install.dir:../../../..}/dist/" regex="solr-cell-\d.*\.jar" />
```
 in the search_api_solr_8.x config sets but it is missing from the search_api_solr_7.x config sets.

**Comment by csandanov** [comment]
I guess that's something that should be reported to search_api_solr 7.x

**Comment by genebobmiller** [comment]
Could be, I'll leave that in your capable hands and look forward to this issue being resolved.

**Comment by csandanov** [comment]
We've made the fix, could you please test the latest version

**Comment by genebobmiller** [comment]
Despite several warnings on the creation of a core such as:

```
Couldn't add files from /opt/solr/server/solr/configsets/search_api_solr_7.x-1.14/../../../contrib/extraction/lib filtered by null to classpath: /opt/solr/server/solr/configsets/search_api_solr_7.x-1.14/../../../contrib/extraction/lib
```

This is now correctly indexing file attachments!

**Comment by csandanov** [comment]
Cannot reproduce, what is the solr version and config set? How do you create a core?

**Comment by genebobmiller** [comment]
From my docker-compose.yml (irrelevant lines removed):
```solr:
    build:
      context: ./wodby-solr
      args:
        BASE_IMAGE_TAG: 7.5.0-alpine
        SOLR_VER: 7.5.0
    environment:
      SOLR_DEFAULT_CONFIG_SET: search_api_solr_7.x-1.14
      SOLR_HEAP: 1024m
```

core created with:
```
make core=wodby -f /usr/local/bin/actions.mk
```
from inside the solr container.

**Comment by csandanov** [comment]
Fixed in 3.1.0
